l10n_br_fiscal/models/document_fiscal_line_mixin_methods.py

```diff
@@ -129,6 +129,50 @@ def fields_view_get(self, view_id=None, view_type="form",
         model_view['fields'] = xfields
         return model_view
 
+    @api.depends(
+        'fiscal_price',
+        'discount_value',
+        'insurance_value',
+        'other_value',
+        'freight_value',
+        'fiscal_quantity',
+        'amount_tax_not_included',
+        'uot_id',
+        'product_id',
+        'partner_id',
+        'company_id')
+    def _compute_amounts(self):
+        for record in self:
+            round_curr = record.currency_id.round
+            # Valor dos produtos
+            record.amount_untaxed = round_curr(
+                (record.price_unit * record.quantity) - record.discount_value)
+
+            record.amount_fiscal = round_curr(
+                (record.fiscal_price * record.fiscal_quantity) -
+                record.discount_value)
+
+            record.amount_tax = record.amount_tax_not_included
+
+            add_to_amount = sum(
+                [record[a] for a in record._add_fields_to_amount()])
+            rm_to_amount = sum(
+                [record[r] for r in record._rm_fields_to_amount()])
```

Comment:
Is it correct to remove the discount_value twice?
![image](https://user-images.githubusercontent.com/9100543/111317392-e271bb80-8642-11eb-9295-5fc8a4930fcd.png)